common/views/components/BetaBar/BetaBar.tsx

```diff
@@ -3,8 +3,9 @@ import { FunctionComponent, ReactElement } from 'react';
 
 const BetaBar: FunctionComponent = (): ReactElement => (
   <MessageBar tagText={'Beta'}>
-    This search tool is in development.{' '}
-    <a href="/works/progress">Find out more</a>.
+    <a href="/works/progress">
```

Comment:
Is another way to do this by having an `aria-label` with the actual text?
[Like the example here](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA8)

Reply:
It is, but think this is more reliable with different screen reader/browser combinations

Reply:
https://www.powermapper.com/tests/screen-readers/labelling/a-aria-label/